GoogleChromeLabs/ps-analysis-tool
# Chore: Remove blocking status and mapping columns from cookies table

## Description

Remove "Blocking Status" and "Mapping" columns from cookies table as they are no longer being used and were be removed in the next version

## Relevant Technical Choices

- Remove "Blocking Status" and "Mapping" columns from `useCookieListing` hook.
- Remove `OrphanedUnMappedInfoDisplay` component which was used for mapping column.
- Remove all translations from `message.json` files.

## Testing Instructions

- Open the cookies table, right-click on the column title to show hidden columns, and check if both columns appear.


## Additional Information:
They were originally added because of the mixed API usage. Some cookies that CDP was able to catch (outbound blocked request cookies) could not be caught by the `webRequest API`.
Due to issues with CDP, we were unsure if the blocking status and mapping problem would resolve. In our testing, we didn't find any blocked cookies. However, since we and QA test a wide variety of websites, we decided to keep these columns for v0.9 in case anyone reports issues with blocking status and cookie mapping information.


## Screenshot/Screencast



---

## Checklist



~- [ ] This code is covered by unit tests to verify that it works as intended.~ NA



Fixes #
